src/components/Cards/CityCard/CityCard.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styles from "./styles.module.scss";
+import { Description } from '../../Description/Description';
+import { Icon_with_Text } from '../../Icon_with_text/Icon';
+import home_icon from "./../../../assets/images/home_icon.svg"
+import rent_icon from "./../../../assets/images/rent_icon.svg";
+import arrow_icon from "./../../../assets/images/areaGuideArrow.svg"
+
+const City_Card = [
```

Comment:
1. data should be in a separate file.
2. naming convention is inconsistent. Should use camelCase.